packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -177,12 +183,18 @@ export class VaRadio {
 
   render() {
     const { label, required, error } = this;
+    const ariaLabel = label + (required ? ' required' : '');
     return (
-      <Host role="radiogroup" aria-invalid={error ? 'true' : 'false'}>
+      <Host
+        role="radiogroup"
+        aria-invalid={error ? 'true' : 'false'}
+        aria-label={ariaLabel}
```

Comment:
Added this `aria-label` so that the screen reader includes the content in the legend

Reply:
@joshkimux do we want to overwrite a potential `aria-label` that a developer added, like:

```jsx
<va-radio
  required
  hint="Some hint text"
  label="Disability type"
  aria-label="Disability type selection" >
   //...
</va-radio>
```

Reply:
Hi @bkjohnson , sorry for coming back to this so late 🙏 
I'm struggling to follow this a little bit as a non-technical person. From what I (think I'm) reading...
- (assumption) [We can't guarantee that the `legend` will be the first child of the fieldset](https://www.accessibility-developer-guide.com/examples/forms/grouping-with-fieldset-legend/#legend-must-be-a-direct-child-of-fieldset) which results in the group not being uniquely labelled
- So Robin's [using an `aria-label` here with a `radiogroup` role to manually label the field](https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role#examples)
- And you're pointing out that a developer _may have already added_ an `aria-label` to the radio group, in which case the `aria-label` Robin is proposing may overwrite the developer's `aria-label`

It's been a while since I looked at this, so my immediate thoughts are:
- do we have any examples of when a developer has already added an `aria-label`? Are these referring to short-term manual fixes made to get around the `legend` not being a first child?
- did we explore perhaps using `aria-labelledby` instead where we reuse the contents of the `legend` to label the `fieldset`? That way if any changes is made to the visible label (including the required string), the aural label will stay consistent without a separate manual entry

This is a weakpoint for me, but maybe someone like @davidakennedy could help here!

Reply:
I think what's being missed in this conversation is that the `aria-label` wasn't previously defined, so the developer could add one if needed. In this work, the `aria-label` is being defined in the line above:

```js
const ariaLabel = label + (required ? ' required' : '');
```

it's taking the existing `label` text (which ends up in the `legend`) and adding `required` to the end if it's required. Brooks is asking if we should be taking away the ability for the developer to add an `aria-label`

Reply:
Robin is right. Do we want these `aria-label` changes to force the `aria-label` to be something like "school required", or do we only want that to be the default which can still be overridden by developers who use the component?

To answer Josh's first question, I couldn't find any examples where a developer was setting `aria-label` on the `<va-radio>` component.

Also, I'm sorry if I dragged you in unnecessarily - I read [this comment](https://github.com/department-of-veterans-affairs/component-library/pull/535#discussion_r977842267) and assumed that the `aria-label` changes were part of a request in a slack DM or something like that :upside_down_face:

Reply:
That's a really really tough question 😮 , I can't think of a scenario where a developer would set an `aria-label` on the `<va-radio>` component, and it would definitely be undesired in most cases from an a11y standpoint. 

I'd rather we encourage folks to make changes to the visible label instead of giving options to folks to use what is basically a band aid (`aria` in a nutshell) to make a fix. Will bite my tongue if someone can think of an actual use case where it would be needed, but in most instances I'd hope folks can work within the `label` and the content/design of a page. @davidakennedy keep me honest here in case I'm missing something!

Reply:
Sounds good - I'll add in a test to serve as documentation of sorts and then get this merged.

Reply:
I agree with @joshkimux here. Let's encourage use of the visible label first. We can always change course if teams being a good use case.